src/index.njk

```diff
@@ -122,11 +122,11 @@ sitemapPriority: 1.0
         <div class="sm:mt-10 md:mt-10 w-full max-w-4xl text-center flex flex-wrap gap-2 justify-around">
             <div class="w-full px-3 mt-4 sm:w-72 md:mt-0 css-full">
                 <div class="w-40 m-auto mb-4">
-                    {% image "./images/pictograms/opensource.png", "Graphics depicting the Open-Source logo.", [160] %}
+                    {% image "./images/pictograms/book-a-demo_red.png", "Icon representing a booked demo.", [160] %}
```

Comment:
This image is for light backgrounds, let me add the right version before merging

Reply:
Ok, feel free to merge right afterwards @Yndira-FlowForge